### ISSUE TITLE
graphql-core-3.2.0 cannot import name 'format_error'

### ISSUE THREAD

**Issue by eMerzh** [graphql-python/graphql-core#156]
I'm just trying to do the upgrade, but i got:

```
File "graphql/controllers.py", line 13, in <module>
from graphql_server import GraphQLParams, GraphQLResponse, HttpQueryError, encode_execution_results, get_graphql_params
File ".venv/lib/python3.10/site-packages/graphql_server/__init__.py", line 15, in <module>
from graphql.error import format_error as format_error_default
ImportError: cannot import name 'format_error' from 'graphql.error' (.venv/lib/python3.10/site-packages/graphql/error/__init__.py)
```

By the way, here is two notes:
1. Many many thanks for the maintenance of this project 👏 
2. not sure ho hard it would be, but it will really help the upgrade to have a comprenhive changelog of what's new, what's possible breaking, and what are the ,new features....  because we might not follow gql.js so closely to be able to tell exactly what's what...

**Comment by Cito** [comment]
Thanks - there is a already a ticket for this [here in graphql-server](https://github.com/graphql-python/graphql-server/issues/85).

Generally, packages using GraphQL-core 3 should always pin the minor version, since it corresponds to the major version of GraphQL.js which can have breaking changes - see also #137.

Any yes, will try to provide more comprehensive changelogs.

**Comment by eMerzh** [comment]
oups sorry... 

thanks for your quick reply

**Comment by Cito** [comment]
This is now solved in graphql-server 3.0.0b5.
